### DISCUSSION TITLE
Server Crash and hight CPU

### DISCUSSION THREAD

**Discussion by Bosterloop** [apohl79/audiogridder#768] [Help]
[Archivio.zip](https://github.com/apohl79/audiogridder/files/7359753/Archivio.zip)
Hi, i have update from 1.2 beta 2 to beta 6 in Logic Pro X
In beta 2 all work good, in beta 6 server crashed andCPU level si very hight, without any plugin insert.
I use Audiogridder in a same machine for AA plugin.
Thanks for your help!

**Comment by Henri75** [comment]
By AA you mean Acustica Audio ?
AG seems to have difficulty to handle those, because they are quite heavy. A lot of issues I have is related to heavy plugin like AA.
Did you enable sandbloxing ? Check that no core is peaking in cpu activity. It could be a load balancing problem between cpu cores.

**Comment by Bosterloop** [comment]
Hi,
 thanks for the reply.
Yes AA is Acustica Audio.
I have a problem about server restart/crash.
The cpu tread is ok and i the last time all work good, except some AA
plugin.
Now many AA plugin have this problem.
I created a discussion inside the official  blog with my files in attached

-- 

Best Regards

Bosterloop Team

-- 




Il ven 22 ott 2021, 10:27 Henri75 ***@***.***> ha scritto:

**Comment by Henri75** [comment]
I fought a bit with AA plugins so may be i can be of help.

Can you precise your problem about restart/ crash ?

Are the DAW and server mac or pc ?

What sample rate are you using ?

Are the AA plugins aquas, n4 or N4 setups ?

Did it stop working after updating AG version ?

If you share description of the exact problem and some logs that would be
help full.

**Comment by Bosterloop** [comment]
Can you precise your problem about restart/ crash ?

I use this example: i load my Logic Pro X project, with AG (fx) in insert
of the channels.

I go to connect all to my server (local or same machine) and all load good.

After some time the server will be restart automatically or not responding,
sometime one time and all work, sometime in loop.

Are the DAW and server mac or pc ?

Mac

What sample rate are you using ?

48Khz

Are the AA plugins aquas, n4 or N4 setups ?

Aquas

Did it stop working after updating AG version ?

In the 1.2 beta 2 all work better, i could load 24 AA instance (server in
the same machine). After the update i can do never of this and the response
of the control knobs on the plugin is "slow" (CPU overload).

I tried to instal the beta 2 again but i can't do work good the old
projects that work perfectly in the past.

I have used metaplugin inside audiogridder for all the projects mentioned
up. for oversample 2X only in export and all working

—

! IMPORTANT !

I have tried to load directly on AG and now all working but obviously I
can’t use the oversampling.

It is possible enable the oversample on audiogridder in the future, like
metaplugin DDMF?


—

Below the retransfer’s link for all my file:

https://we.tl/t-d554cL5dZL

—


Thanks a lot for your time!
